src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -192,6 +192,12 @@ Copyright (c) .NET Foundation. All rights reserved.
         <RelativePath>@(_DebugSymbolsIntermediatePath->'%(Filename)%(Extension)')</RelativePath>
       </ResolvedFileToPublish>
 
+      <!-- Copy the documentation file (.xml), if any -->
+      <ResolvedFileToPublish Include="@(_DebugSymbolsIntermediatePath)"
```

Comment:
I think we probably already have tests which would cover this.  If DebugSymbolsIntermediatePath were changed to FinalDocFile here, I think we would see some publish tests failing as there would be a new file in the publish output that the tests aren't expecting.